<launch>

	<!-- <include file="$(find folding_vision)/launch/vision.launch" /> -->
	<node pkg="folding_geometry" type="generate_poly_from_points.py" name="poly_maker" output="screen">	
		<remap from="input_3d_points" to="stereo_points_3d" />
		<remap from="output_stamped_polygon" to="poly_maker_output" />
		<param name="poly_frame_name" value="laundry_frame" />
		<param name="world_frame" value="base_footprint" />
	</node>
	
        <!--<node pkg="folding_geometry" type="poly_gui_bridge.py" name="poly_gui_bridge" output="screen">  -->
	<node pkg="folding_geometry" type="folding_main.py" name="folding_main" output="screen"> 
		<remap from="input" to="poly_maker_output" />
		<remap from="trajectory_output" to="fold_trajectory" />
		<param name="mode" value="towel" />
	</node>		
    
	<include file="$(find stereo_click)/launch/wide_stereo_click.launch" /> 
	<include file="$(find pr2_simple_arm_motions)/launch/move_arms.launch"/>
	<include file="$(find pr2_simple_torso_motions)/launch/move_torso.launch"/>
        <include file="$(find pr2_simple_head_motions)/launch/move_head.launch"/>
        <include file="$(find pr2_simple_base_motions)/launch/move_base.launch"/>

	<include file="$(find unfolding_smach)/launch/stances.launch" />
	
	<!--
	<include file="$(find pr2_arm_kinematics)/launch/pr2_ik_rarm_node.launch" />
	<include file="$(find pr2_arm_kinematics)/launch/pr2_ik_larm_node.launch" />
	-->
	<include file="$(find pr2_costs)/launch/move_arm_costs.launch" />
	<!--<include file="$(find gpp_navigation)/launch/station_nav.launch"/>-->
	
</launch>
